sparse_ho/models/lasso.py

```diff
@@ -385,8 +386,12 @@ def get_hessian(X, y, mask, dense, log_alpha):
             Logarithm of hyperparameter.
         """
         X_m = X[:, mask]
-        hessian = X_m.T @ X_m  # TODO no normalization by n_samples?
-        return hessian
+        n_samples, size_supp = X_m.shape
+
+        def mv(v):
+            return X_m.T @ (X_m @ v) / n_samples
+        linop = LinearOperator((size_supp, size_supp), matvec=mv)
```

Comment:
in the docstring you need a comment that you use this function to not lose the computational speedups of having X sparse

Reply:
done